gui/README.md

```diff
@@ -22,3 +22,10 @@ Point your browser to
 Substitute:
 * `[namespace]` for the wf's namespace.
 * `[wf-name]` for the wf's name.
+
+## Kubectl proxy
+When this GUI is used in combination with kubectl proxy, run the following proxy to deal with CORS:
```

Comment:
'When it is used in combination with...' sounds like there is an alternative, how do people use it without kubectl or  is the corsproxy required?